src/components/hide.js

```diff
@@ -0,0 +1,14 @@
+import styled from '@emotion/styled';
+import { Box, get } from 'theme-ui';
+
+const Hide = styled(Box)`
+  ${({theme, when}) => {
+    return `
+      @media Screen and (max-width: ${get(theme, 'breakpoints')[when]}) {
+        display: none;
+      }
+    `;
+  }}
+`;
+
```

Comment:
Think about the case where we want to hide a block in desktop, not in mobile devices.